CRM/Utils/File.php

```diff
@@ -302,7 +305,7 @@ public static function runSqlQuery($dsn, $queryString, $prefix = NULL, $dieOnErr
         $db = DB::connect($dsn, $options);
       }
       catch (Exception $e) {
-        die("Cannot open $dsn: " . $e->getMessage());
+        throw new CRM_Core_Exception("Cannot open $dsn: " . $e->getMessage());
```

Comment:
Not even sure why we're catching and re-throwing this.

Reply:
I feel like this was possibly something maybe historical maybe legacy to do with install

Reply:
Good question. One I don't really have an answer to. 

This PR's literally the lightest touch change I could make - I'm slightly fearful of what might happen upstream.

Are you suggesting I strip out the `try/catch` entirely?